### PR TITLE
improve /profit to not raise an exception if no trade is closed

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
/Profit should not return 502 just because no trade has been closed.
Instead, it should return the profit for open trades (or no profit, if no trade exists).

## Quick changelog

- Improve telegram output for just started bots (when calling /profit)
- improve output for API (no 502 - but results instead)

## What's new?
No change to /profit when closed trades exist

shows "No trades yet." if no trade was yet opened.

2020-05-29_09h44_14
2020-05-29_09h44_24

### CODE REVIEW

File: freqtrade/rpc/telegram.py [hunk 1]
```diff
@@ -311,38 +311,40 @@ def _profit(self, update: Update, context: CallbackContext) -> None:
...
+        profit_all_fiat = stats['profit_all_fiat']
+        trade_count = stats['trade_count']
+        first_trade_date = stats['first_trade_date']
+        latest_trade_date = stats['latest_trade_date']
+        avg_duration = stats['avg_duration']
+        best_pair = stats['best_pair']
+        best_rate = stats['best_rate']
+        # Message to display
+        if stats['closed_trade_count'] > 0:
+            markdown_msg = ("*ROI:* Close trades\n"
```
Comment (hroff-1902) [inline freqtrade/rpc/telegram.py:331]:
```suggestion
            markdown_msg = ("*ROI:* Closed trades\n"
```

**Comment by hroff-1902** [comment]
Maybe it's better to write in the case of no trades at all:
"No, open trades. No closed trades."
or
"No, open trades."
?

**Comment by xmatthias** [comment]
> Maybe it's better to write in the case of no trades at all:
> "No, open trades. No closed trades."
> or
> "No, open trades."
> ?

every option with a comma sounds very much off to me.

also - i didn't want to completely change this message - just remove the unnecessary exception as it's giving problems for the API.

File: freqtrade/rpc/telegram.py (hunk 1 above)
Comment (xmatthias) [reply freqtrade/rpc/telegram.py]: bu can't commit from here - would break tests :D

**Comment by hroff-1902** [comment]
> > Maybe it's better to write in the case of no trades at all:
> > "No, open trades. No closed trades."
> > or
> > "No, open trades."
> > ?
> 
> every option with a comma sounds very much off to me.
> 
> also - i didn't want to completely change this message - just remove the unnecessary exception as it's giving problems for the API.

ah, read above as "No open trades. No closed trades." or "No open trades.", I do not know where that comma came from.
Actually, "No trades yet" is also okay...

**Pull request merged**